### PR TITLE
Very important note about URL format

### PR DESCRIPTION
**Pull request by xperiandri**
I spent 4 hours finding what is wrong!
If you can you must highlight it with red font

### CODE REVIEW

File: aspnetcore/security/cors.md [hunk 1]
```diff
@@ -59,8 +59,7 @@ You can specify a cross-origin policy when adding the CORS middleware using the
 
 [!code-csharp[Main](cors/sample/CorsExample1/Startup.cs?highlight=11,12&range=22-38)]
 
-> [!NOTE]
-> URL must be specified without trailing slash otherwise comparison will return false and no header will be returned.
+**Note:** The URL must be specified without trailing slash. If the URL terminates with `/`, the comparison will return `false` and no header will be returned.
```
Comment (badersur) [inline aspnetcore/security/cors.md]: Shouldn't it be: without "a" trailing slash?

Comment (badersur) [reply aspnetcore/security/cors.md]: I didn't notice your second commit!

Comment (guardrex) [reply aspnetcore/security/cors.md]: Present tense version:

> The URL is specified without trailing slash. If the URL terminates with `/`, the comparison returns `false` and no header is returned.

**Comment by Rick-Anderson** [comment]
@xperiandri  thanks for the note.
@badersur  thanks for the help.

**Pull request merged**